src/sync.d

```diff
@@ -2171,7 +2236,7 @@ final class SyncEngine
 			break;
 		case ItemType.dir:
 		case ItemType.remote:
-			log.log("Creating local directory: ", path);
+			log.log("Creating local directory: ", "./" ~ path);
```

Comment:
I'm a bit surprised about this one here. Is this necessary? Do we chdir into the respective directories?

Reply:
With this one - when creating local directories, the following was being printed:
```
Creating local directory: level 1/level 2
Creating local directory: Images/child
Creating local directory: Images/Child 34
```
With this PR, that now becomes:
```
Creating local directory: ./level 1
Creating local directory: ./shared
Skipping item - excluded by skip_dir config match: /Images
Skipping item - excluded by skip_dir config match: /Music
Creating local directory: ./level 1/level 2
Creating local directory: ./level 1/level 2/level 3
```
It was more so output consistency .. can remove this as really non essential - will do so then merge the PR.